scripts/prereqs/install_prereqs.sh

```diff
@@ -27,7 +27,7 @@ set -e
 #  $1 - 'full' if build prerequisites should be installed, 'release' if only runtime prerequisites should be installed
 
 OPENSSL_VERSION=${OPENSSL_VERSION:-1.1.1o}
-MAVEN_VERSION=${MAVEN_VERSION:-3.9.1}
```

Comment:
I think we should also change the maven link to use archive to download -- otherwise, we might fail on this as newer versions come out and apache retires the link